app/Nova/Filters/UserAccessActive.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter,SlevomatCodingStandard.Functions.UnusedParameter,SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint
+
+namespace App\Nova\Filters;
+
+use Illuminate\Http\Request;
+use Laravel\Nova\Filters\Filter;
+use Illuminate\Database\Eloquent\Builder;
+
+class UserAccessActive extends Filter
+{
+    /**
+     * The displayable name of the action.
+     *
+     * @var string
+     */
+    public $name = 'Access Active';
+
+    /**
+     * Apply the filter to the given query.
+     *
+     * @param \Illuminate\Http\Request  $request
+     * @param \Illuminate\Database\Eloquent\Builder  $query
+     * @param string  $value
+     *
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function apply(Request $request, $query, $value): Builder
```

Comment:
Can you type hint this?

Reply:
No, it seemed to not be valid since it doesn't match the parent signature.

Reply:
Ah right that's a thing that happens.